recipes/example/ruby/meta.yaml

```diff
@@ -85,8 +85,8 @@ about:
 
 extra:
   recipe-maintainers:
-    # Feel free to keep me in the recipe-maintainers list for ruby packages.
-    - sodre
+    # Keep the ruby group in the maintainers list
+    - conda-forge/ruby
```

Comment:
@conda-forge/ruby, I would like to have the `ruby` recipes have a common group of maintainers as it was done for the `r` packages. I would like to know if you guys are okay with it (@scopatz, @jimmynguyc)? If not, are you okay with us dropping you from the @conda-forge/ruby team?